Source/Core/Core/PowerPC/JitLLVM/JitLLVM_Tables.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Core/PowerPC/Gekko.h"
+#include "Core/PowerPC/PPCTables.h"
+#include "Core/PowerPC/JitLLVM/Jit.h"
```

Comment:
Why? If that's the case then may as well as change every recompiler's tables file.